WooCommerce.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -84,7 +84,7 @@
       },
       {
         "package": "swift-numerics",
-        "repositoryURL": "https://github.com/apple/swift-numerics",
+        "repositoryURL": "https://github.com/apple/swift-numerics.git",
```

Comment:
I accidentally pushed a diff of the file that removed all the dependencies when rebasing the branch, and then I tried to revert it but ended up with this change. I'm not sure how to revert this, but I hope this is not a big problem because CI can still resolve the dependencies with this. WDYT?